Team_Roster/pages/depthChart.py

```diff
@@ -3,6 +3,7 @@
 import sqlalchemy as sa
 from app.models import People, Fielding, Batting, Team, Pitching, Season
 from sqlalchemy import func, and_
+from pages.stats import *
```

Comment:
Move the functions back in here, please :)
Otherwise, other changes made here will have crazy merge conflicts. It was good thinking, but was implemented too late to be worthwhile significantly changing how everything is set up.